components/event-bus/test/integration/publish_push_test.go

```diff
@@ -279,3 +278,58 @@ func Test_pushRequestShouldNotIncludeKymaTopicHeader(t *testing.T) {
 		t.Fatalf("request to endpoint should not include the header %s", headerKymaTopic)
 	}
 }
+
+func Test_sameSubjectSubscribersInDifferentNamespacesShouldReceiveEventsOfThatSubject(t *testing.T) {
```

Comment:
I think this test is indecisive. The test case we need to verify is the isolation of two subscriptions with the same name in two different namespaces. So either send an event to sub1 and verify that sub 2 didn't receive it, or send two different events to the two subscriptions and verify each received the correct one.

The current test will not fail if the bug was not fixed so it does not expose it.

Reply:
You can keep the current test, but the crucial one is the isolation test IMHO.

Reply:
As per our offline discussion with @Abd4llA, this test is meant to verify that two subscriptions with the same name but in different kubernetes namespaces, can both receive the same events when subscribing to the same subject.

As agreed with @Abd4llA, no more action required regarding this and it is resolved.